qaframework-bdd-tests/src/test/resources/features/refapp-2.x/09-form/forms.feature

```diff
@@ -1 +1,12 @@
-Feature: Forms Management
\ No newline at end of file
+Feature: Form Management
+
+  Background:
+  Given  a user click on manage html forms
+
+  @selenium
+  @forms
```

Comment:
I see from the selenium implementation, there is one form that is being populated with what a user can do; add a form, edit the form and deleting the form, 

isn't the tag supposed to be `@form`?